src/eko/anomalous_dimensions/as1aem1.py

```diff
@@ -0,0 +1,412 @@
+                -32
+                + N
+                * (
+                    -8 * (1 + np.pi**2)
+                    + N
+                    * (
+                        64 * g3n * (1 + N) ** 3
+                        - 3 * (3 + N) * (3 + N**2)
+                        - 8 * (2 + N) * np.pi**2
+                    )
+                )
+            )
+        )
+        / (N**3 * (1 + N) ** 3)
+        + 16.0 / 3 * np.pi**2 * S1p1h
+        - 4 * S3h
+        - 32 * S3
+        + 4 * S3p1h
+        - 16 * zeta3
+    )
+    return constants.CF * result
+
+
+@nb.njit("c16(c16,u1,c16[:])", cache=True)
+def gamma_nsm(N, nf, sx):
+    """
+    Computes the O(as1aem1) singlet-like non-singlet anomalous dimension.
+
+    Implements Eq. (3.5) of :cite:`Moch:2004pa`.
+
+    Parameters
+    ----------
+        N : complex
+            Mellin moment
+        nf : int
+            Number of active flavors
+        sx : np array
+            List of harmonic sums
+
+    Returns
+    -------
+        gamma_nsm : complex
+            O(as1aem1) singlet-like non-singlet anomalous dimension
+            :math:`\\gamma_{ns,-}^{(1,1)}(N)`
+    """
+    S1 = sx[0]
+    S2 = sx[1]
+    S3 = sx[2]
+    S1h = harmonics.harmonic_S1(N / 2)
+    S2h = harmonics.harmonic_S2(N / 2)
+    S3h = harmonics.harmonic_S3(N / 2)
+    S1p1h = harmonics.harmonic_S1((N + 1.0) / 2)
+    S2p1h = harmonics.harmonic_S2((N + 1) / 2)
+    S3p1h = harmonics.harmonic_S3((N + 1) / 2)
+    g3n = harmonics.mellin_g3(N)
+    zeta2 = harmonics.zeta2
+    zeta3 = harmonics.zeta3
+    result = (
+        -32.0 * zeta2 * S1h
+        - 8.0 / (N + N**2) * S2h
+        + (24 + 16 / (N + N**2)) * S2
+        + 8.0 / (N + N**2) * S2p1h
+        + S1
+        * (
+            -16 * (3 / N**2 - 3 / (1 + N) ** 2 - 2 * zeta2)
+            + 16 * S2h
+            - 32 * S2
+            - 16 * S2p1h
+        )
+        + (
+            72
+            + N
+            * (
+                96
+                - 3
+                * N
+                * (8 + 64 * g3n * N * (1 + N) ** 3 + 3 * N * (3 + N) * (3 + N**2))
+                + 8 * N * (1 + N) ** 2 * np.pi**2
+            )
+        )
+        / (3.0 * N**3 * (1 + N) ** 3)
+        + 16.0 / 3 * np.pi**2 * S1p1h
+        + 4 * S3h
+        - 32 * S3
+        - 4 * S3p1h
+        - 16 * zeta3
+    )
+    return constants.CF * result
+
+
+@nb.njit("c16(c16,u1,c16[:])", cache=True)
+def gamma_nsV(N, nf, sx):
+    """
+    Computes the O(as1aem1) valence non-singlet anomalous dimension.
+
+    Implements Eq. (3.5) of :cite:`Moch:2004pa`.
+
+    Parameters
+    ----------
+        N : complex
+            Mellin moment
+        nf : int
+            Number of active flavors
+        sx : np array
+            List of harmonic sums
+
+    Returns
+    -------
+        gamma_nsV : complex
+            O(as1aem1) singlet-like non-singlet anomalous dimension
+            :math:`\\gamma_{ns,V}^{(1,1)}(N)`
+    """
+
+    return gamma_nsm(N, nf, sx)
```

Comment:
So far we dealt with the collapse of sectors on a higher level (to avoid recomputation) - I would keep that scheme and drop this function